File: demo/src/main/java/com/bluelinelabs/conductor/demo/changehandler/ArcFadeMoveChangeHandlerCompat.java
package com.bluelinelabs.conductor.demo.changehandler;

import com.bluelinelabs.conductor.changehandler.TransitionChangeHandlerCompat;

public class ArcFadeMoveChangeHandlerCompat extends TransitionChangeHandlerCompat {

    public ArcFadeMoveChangeHandlerCompat() {
        super(new ArcFadeMoveChangeHandler(), new ArcFadeMoveChangeHandlerTransitionsEverywhere());
    }

}

File: demo-kotlin/src/main/java/com/bluelinelabs/conductor/demo/kotlin/changehandler/ArcFadeMoveChangeHandlerCompat.kt
package com.bluelinelabs.conductor.demo.kotlin.changehandler

import com.bluelinelabs.conductor.changehandler.FadeChangeHandler
import com.bluelinelabs.conductor.changehandler.TransitionChangeHandlerCompat

class ArcFadeMoveChangeHandlerCompat : TransitionChangeHandlerCompat {

    constructor() : super() {}

    constructor(vararg transitionNames: String) : super(ArcFadeMoveChangeHandler(*transitionNames), FadeChangeHandler())

}
